doc/overview.rst

```diff
@@ -0,0 +1,12 @@
+=======================
+Overview of dbxincluder
+=======================
+
+dbxincluder is an implementation of the XInclude 1.1 specification (https://www.w3.org/TR/xinclude-11)
+with support for DocBook transclusion (http://docbook.org/docs/transclusion).
+
+
+Limitations
+===========
+
+It does not support the XInclude 1.1 "xpointer" attribute, as the XPointer functionality is currently not exposed by lxml.
```

Comment:
I don't think this statement is completely true. The `xi:include` element is just an element which belongs to a namespace. The lxml library exposes this, if you don't resolve any XInclude elements. By the way, `xpointer` attribute was also available in version 1.0.

So I would just write:

    The XInclude "xpointer" attribute is currently not supported.